shoes-core/lib/shoes/text.rb

```diff
@@ -17,6 +17,10 @@ def app
       parent && parent.app
     end
 
+    def links
+      Shoes::Link.find_links(@texts)
+    end
```

Comment:
Yeah, the fact that we needed similar finding between the `TextBlock` (full Shoes element) and the `Text` "segment" classes which aren't real Shoes elements, makes this awkward.

We've definitely stuck more to the module based approach in Shoes, so although I'm not 100% a fan, that'd probably be more expected. I'll give it a try.